src/components/chat-item/prompt-input/prompt-text-input.ts

```diff
@@ -72,6 +73,9 @@ export class PromptTextInput {
         },
         focus: () => {
           this.render.addClass('input-has-focus');
+          if (typeof this.props.onFocus !== 'undefined') {
```

Comment:
Even though type val check is sufficient, `this.props.onFocus != null` would be better i suppose.

Reply:
Not mandatory btw, just a nit for later.